src/App.css

```
@@ -1,10 +1,10 @@
 * {
-  box-sizing: border-box;
-  margin: 0;
-  padding: 0;
   font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
     'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
     sans-serif;
```

Here we can already set our font globally

```suggestion
  font-family: 'Montserrat', sans-serif;
```

---

Sounds like a good idea, I was thinking the same thing but forgot to change it, thanks!